tests/integration/test_summary.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import sqlite3
+from contextlib import closing
+
+import pytest
+
+from airbase.fetch import fetch_json
+from airbase.resources import E1A_SUMMARY_URL
+from airbase.summary.db import summary
```

Comment:
Could you add one additional test that the `summary.sqlite` file actually exists? I can foresee it getting deleted by accident in the future and that will help surface the issue more quickly

Reply:
Alternately feel free to give me access to your fork and I'll be happy to add it myself

Reply:
the module fails to load if the DB file is missing

```
ImportError while loading conftest '/home/alvarov/Python/airbase/tests/conftest.py'.
tests/conftest.py:6: in <module>
    import airbase
airbase/__init__.py:3: in <module>
    from .airbase import AirbaseClient, AirbaseRequest
airbase/airbase.py:15: in <module>
    from .summary import DB
airbase/summary/__init__.py:1: in <module>
    from .db import DB
airbase/summary/db.py:16: in <module>
    class DB:
airbase/summary/db.py:21: in DB
    db = sqlite3.connect(f"file:{summary()}?mode=ro", uri=True)
airbase/summary/db.py:12: in summary
    with resources.path(__package__, "summary.sqlite") as path:
/usr/lib/python3.8/contextlib.py:113: in __enter__
    return next(self.gen)
E   FileNotFoundError: [Errno 2] No such file or directory: '/home/alvarov/Python/airbase/airbase/summary/summary.sqlite'
```

Reply:
Ah of course, since the connection is established at import time. That makes sense since it's essential for the library to function, and the error message is clear. Works for me!